molecule/identity/molecule.yml

```diff
@@ -0,0 +1,47 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: identity-instance
+    image: $MOLECULE_ROLE_IMAGE
+    dockerfile: ../../tests/molecule/Dockerfile-noprivs.j2
+    command: "/lib/systemd/systemd"
+    privileged: true
+    tmpfs:
+      - /run
+      - /run/lock
+      - /tmp
+    volume_mounts:
+      - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
+    groups:
+      - database
+      - activemq
+      - repository
+      - trusted_resource_consumers
+      - identity
+      - nginx
+    published_ports:
+      - 80/tcp
+      - 0.0.0.0:443:443/tcp
+      - 0.0.0.0:8080:8080/tcp
```

Comment:
any reason why port 80 is mapped dynamically and not others?

Reply:
mostly copy paste from similar suites